### PR TITLE
[How to source-git?] update

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: content/docs/how-to-source-git.md [hunk 1]
```diff
@@ -182,6 +182,17 @@ bcc2c8a292 resolved: create /etc/resolv.conf symlink at runtime
 1d39b39df9 Revert "journald: periodically drop cache for all dead PIDs"
 ```
 
+In case the patches are not in git format you need to apply them with `patch`:
+```
+$ patch -p1 --verbose --fuzz=0 <fedora/some-fix.patch
```
Comment (lachmanfrantisek) [inline content/docs/how-to-source-git.md]: What about removing those patches from the specfile?

File: content/docs/how-to-source-git.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ from the systemd-stable repo. We'll start a new git branch in our repo named
 `239-sg` (sg as source-git):
 
 ```bash
-$ git checkout -B 239-sg 8bca4621fc003a148c70248c55aa877dfe61fd3f
+$ git switch -C 239-sg 8bca4621fc003a148c70248c55aa877dfe61fd3f
```
Comment (lachmanfrantisek) [inline content/docs/how-to-source-git.md]: I never played with this command. Looks good.
```
       THIS COMMAND IS EXPERIMENTAL. THE BEHAVIOR MAY CHANGE.
```

File: content/docs/how-to-source-git.md (hunk 1 above)
Comment (jpopelka) [reply content/docs/how-to-source-git.md]: Not sure I understand. From downstream or upstream specfile? Why?

Comment (lachmanfrantisek) [reply content/docs/how-to-source-git.md]: Probably from both:
- in the source-git, the changes should in a form of commits
- in the dist-git, the changes should be in the form of auto-generated patches

**Comment by jpopelka** [comment]
PTAL @lachmanfrantisek

File: content/docs/how-to-source-git.md [hunk 3]
```diff
@@ -168,26 +155,60 @@ $ git commit -m 'add fedora packaging'
 
 ### Applying downstream patches
 We are getting to the core of source-git: we work with code in it, not with
-patches. Hence we need to apply the downstream patches:
+patches. But first we need to instruct packit from which point in history start
+downstream patches.
+Add [upstream_ref](https://packit.dev/docs/configuration/#upstream-ref) key into
+`.packit.yaml` (well, there's currently no one in systemd upstream):
```
Comment (lachmanfrantisek) [inline content/docs/how-to-source-git.md]: ?

**Pull request merged**